.circleci/config.yml

```
@@ -1,5 +1,16 @@
 version: 2.1
 
+commands:
+    check-version:
+        description: Halt Circle CI step if pkg version did not change
+        steps:
+            - run: |
+                version_diff=$(git diff main | grep \"version\": | wc -l || :)
```

```suggestion
                version_diff=$(git diff main package.json | grep \"version\": | wc -l || :)
```
I'd restrict the diff to package.json only to avoid false positives in case the string `"version":` appears in the diff from some other place in the repository.

---

Agree 👍